actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Fix `ActionController::Parameters==`
+
```

Comment:
I'd say we just remove it altogether. It's fixing the new unreleased parameters, not the one from 4.2.